src/newSeed/stage4.ts

```diff
@@ -91,6 +91,8 @@ export class Stage4 extends BaseStage {
       message = "Please enter a non-zero value for the Funding Target";
     } else if (!this.seedConfig.seedDetails.fundingMax || this.seedConfig.seedDetails.fundingMax === "0") {
       message = "Please enter a non-zero number for the Funding Max";
+    } else if (Number.parseInt(this.seedConfig.seedDetails.fundingTarget) > Number.parseInt(this.seedConfig.seedDetails.fundingMax)) {
```

Comment:
Wei values can overflow javascript numbers.  You need to use bignumbers here.

```
BigNumber.from(this.seedConfig.seedDetails.fundingTarget).gt(this.seedConfig.seedDetails.fundingMax)
```

Reply:
Okay